hphp/runtime/base/runtime-option.h

```diff
@@ -606,6 +606,12 @@ struct RuntimeOption {
   F(int64_t, PerfWarningSampleRate, 1)                                  \
   /* Profiling flags */                                                 \
   F(bool, EnableReverseDataMap, false)                                  \
+  /* PPC64 Option: Minimum immediate size to use TOC */                 \
+  F(uint16_t, PPC64minTOCImmSize, 64)                                   \
+  /* PPC64 Option: Dump the number of elements stored in TOC */         \
+  F(bool, PPC64dumpTOCNelements, false)                                 \
```

Comment:
This should be a `Trace` module instead of a RuntimeOption, and you should use `FTRACE()` or similar for your dumping—see `util/trace.h`.